### DISCUSSION TITLE
Update plugin: Collapsible Modern Chat

### DISCUSSION THREAD

**Discussion by bearryj** [runelite/runelite#18145] [Ideas]
Please, add the option to be able to keybind the chat between hidden and unhidden. #18145

**Comment by bearryj** [comment]
plugins/collapsible-modern-chat

**Comment by Felanbird** [comment]
Support for hub plugins should be directed to the author of the plugin. You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.
